deploy/bz1883589-kubeapiserver-revert/00-service-accounts.flowschema.patch.yaml

```diff
@@ -1,7 +0,0 @@
-apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
-kind: FlowSchema
-name: service-accounts
-applyMode: AlwaysApply
-# https://bugzilla.redhat.com/show_bug.cgi?id=1883589#c18
-patch: '{"spec":{"priorityLevelConfiguration":{"name":"workload-low"}}}'
```

Comment:
Does this need to be applied as a part of this to set the value back to the shipped default?

Reply:
My intent here was to remove and previous P&F patches applied on the clusters of which the one from BZ https://bugzilla.redhat.com/show_bug.cgi?id=1883589 was amongst them. This BZ was used to track the issue initially and that patch was proposed. However, that didn't seem to work and it further got fixed in 4.5.30 and 4.6.13 as per https://bugzilla.redhat.com/show_bug.cgi?id=1908383#c19. This [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1908383) mentions of the same issue even after upgrading to 4.5.16 so better to remove that patch.

I did not want to mix two patches and wanted to consider the common patch for 4.5 and 4.6 which was mentioned more recently as a workaround till the customer upgrades the cluster. Let me know if this answers your query.

Reply:
By removing the patch from hive it simply stops applying the patch.  If by "remove" you mean "revert" then you have to patch in the old value as was prepared in `bz1883589-kubeapiserver-revert`.  As it stands any cluster that has the flowschema patch from `bz1883589-kubeapiserver-fix` will continue to have that patch.  That leaves pre-existing clusters inconsistent with new clusters.